apps/web/src/views/Farms/v4/components/PoolsPage.tsx

```diff
@@ -9,16 +10,23 @@ import {
   SubMenu,
   ITableViewProps,
   TableView,
+  FeeTier,
 } from '@pancakeswap/uikit'
+import { TokenPairImage } from 'components/TokenImage'
 import { useTranslation } from '@pancakeswap/localization'
+import { ERC20Token } from '@pancakeswap/sdk'
+import { TokenOverview } from '@pancakeswap/widgets-internal'
+import { useFarmsV3WithPositionsAndBooster } from 'state/farmsV3/hooks'
 import { PoolsFilterPanel } from './PoolsFilterPanel'
 
 interface IDataType {
   name: string
-  feeTier: number
-  apr: number
-  tvl: number
+  feeAmount: number
+  cakeApr: number
+  activeTvlUSD: number
```

Comment:
why change to `active`

Reply:
because type have limit, so there should be change to these key to ensure the deploy, it's ok, just mock data, i'll replace to real data in the future, so ignore them to ensure the page can preview now